### ISSUE TITLE
[Accessibility] The narrator announcement of content is confusing when using a mouse to increase NumericUpDown value

### ISSUE THREAD

**Issue by Jessie-Zhang01** [dotnet/winforms#3453]
* .NET Core Version: 
.Net 5.0.100-preview.7.20314.1 from 5.0 master branch

* Have you experienced this same bug with .NET Framework?: 
Yes

**Minimal repro:**

1. Extract “Private_1588_LTR_without_ComboBox.zip” and open in VS, then build and run this application.
2. Start Narrator and focus on NumericUpDown, using mouse to increase NumericUpDown value and observe Narrator behavior.
[Private_1588_LTR_without_ComboBox.zip](https://github.com/dotnet/winforms/files/4810722/Private_1588_LTR_without_ComboBox.zip)

**Expected behavior:**
The narrator should announces the correct content when using mouse to increase NumericUpDown value.

**Actual behavior:**
The narrator announces content is confusion when using mouse to increase NumericUpDown value.
![image](https://user-images.githubusercontent.com/45864649/84763354-a13e7380-affe-11ea-8f6c-0f2af0a7902e.png)
![Numeric_increase_by_mouse](https://user-images.githubusercontent.com/45864649/84762934-05146c80-affe-11ea-920b-7bff9f198e62.gif)

**Comment by merriemcgaw** [comment]
@Jessie-Zhang01 can you show us what Inspect is doing with this?

**Comment by Jessie-Zhang01** [comment]
@merriemcgaw about Inspect testing, Inspect behavior is fine, no issue was found.
The following image is from using the mouse to increase the NumericUpDown value.
![image](https://user-images.githubusercontent.com/45864649/85943606-b6eb5b80-b963-11ea-9e3c-d1a4b0c0a42c.png)
The following image is from increasing the NumericUpDown value using the keyboard.
![image](https://user-images.githubusercontent.com/45864649/85943617-c10d5a00-b963-11ea-9c62-f1b13a7127e4.png)

**Comment by merriemcgaw** [comment]
Since we are providing the right data to UIA and mouse interaction with the controls is not the primary scenario of Narrator, I think we're going to go ahead and close this issue.
